raddb/mods-available/krb5

```diff
@@ -1,76 +1,145 @@
 # -*- text -*-
 #
+# :toc:
+#
 #  $Id$
 
+#######################################################################
+#
+#  = Kerberos Module
+#
+#  The `krb5` module enables the use of Kerberos 5 for authentication.
+
+#
+#  ## Default instance
+#
+
 #
-#  Kerberos.  See doc/modules/rlm_krb5 for minimal docs.
+#  krb5 { ... }::
```

Comment:
If doc/modules/rlm_krb5 still exists it'd be good to merge its contents in here.

Reply:
Maybe that's what's already at the bottom :)